webcommon/web.inspect/src/org/netbeans/modules/web/inspect/ui/CssStylesPanelProviderImpl.java

```diff
@@ -139,7 +139,6 @@ private void initNoStylesLabel() {
         noStylesLabel.setHorizontalAlignment(SwingConstants.CENTER);
         noStylesLabel.setVerticalAlignment(SwingConstants.CENTER);
         noStylesLabel.setEnabled(false);
-        noStylesLabel.setBackground(new BeanTreeView().getViewport().getView().getBackground());
```

Comment:
I tested this on current master - the original code works and is not broken. I admit, that it is questionable, but it works ;-)

From my perspective it would be good to either guard this with a try-catch or with a null check, with a comment, that the null case happens in test variant. (comment matches all cases)

Reply:
where is that window?

Reply:
- Create a new "HTML5/JS Application"
- Add ` <style>div { color: firebrick }</style>` inside the `head` element of the sample `index.html`
- Choose "Embedded WebKit Browser": 
![image](https://user-images.githubusercontent.com/2179736/176937316-f668db89-540b-4f46-9120-06f53c20ff6a.png)
- Run the project:
![image](https://user-images.githubusercontent.com/2179736/176937403-649fd6e1-5c7a-4292-a603-43f0c41de586.png)
- The CssStylesPanel is show in the top right: 
![image](https://user-images.githubusercontent.com/2179736/176939613-40b13922-c0dc-4755-aeb5-e3778a8daea4.png)

Reply:
thanks @matthiasblaesing, I would have never found that panel.

I pushed an alternative proposal for the NPE fixes. I tested it on several LAFs and all had the correct colors.

And instead of testing on 18, I leave it at 19-ea but disable the xserver there instead, since I believe we get more value when testing on 19ea where we can detect issues early.